docs/prerequisites.md

```diff
@@ -0,0 +1,31 @@
+# Prerequisites
```

Comment:
I would expect this file to list the external dependencies to build and run the benchmarks.